docs/guide/dev/features/output_format/README.md

```diff
@@ -82,7 +82,7 @@ this approach used when creating an editor instance:
 
 Another solution is to use the {@linkapi CKEDITOR } object which will cause all editor instances to be changed:
 
-	{@linkapi CKEDITOR.on CKEDITOR.on}( 'instanceReady', function( ev ) {
+	{@linkapi CKEDITOR.event#on CKEDITOR.on}( 'instanceReady', function( ev ) {
```

Comment:
Technically this is just a mitigation but not a fix to a source of the problem.

The source of this issue is that there's no API entry for `CKEDITOR.on()` function (and other members contributed by event mixin).

`CKEDITOR` namespace [**does inherit event mixin**](https://github.com/ckeditor/ckeditor-dev/blob/c199bf1/core/ckeditor_basic.js#L14) so API docs should reflect that.

To make it short: the proper fix is to correct CKEditor API docs, so that `CKEDITOR` namespace implements `CKEDITOR.event` mixin. And revert this workaround.